CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for investing your time in contributing to our project!
+
+Read our [Code of Conduct](https://github.com/vektra/mockery/blob/master/CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+## Local development setup
+
+This project uses [Taskfile](https://taskfile.dev/), please see the [Install Taskfile](https://taskfile.dev/installation/) and
```

Comment:
You shouldn't have to reference the installation docs for taskfile, we actually import all tools mockery uses in the tools/ sub-module so you can just do `go mod download -x` to get everything.

Reply:
It just downloads all the dependencies listed in go.mod. The tools submodule contains the tools in its go.mod file so asking go to download the deps should take care of it for you. 

No you can keep it, it's still useful.